prod-devel-rcar.yaml

```diff
@@ -104,7 +104,7 @@ common_data:
     - [PACKAGE_CLASSES, "package_ipk"]
 
     # Additional testing and debug tools
-    - [IMAGE_INSTALL:append, " expect ltrace strace evtest"]
+    - [IMAGE_INSTALL:append, " expect ltrace evtest"]
```

Comment:
> If we will need this tool, we will investigate this issue later on.

I believe, it is the best time to investigate it :)

Or at least - create an issue on GitHub.

Reply:
Hi @lorc,

I'm already deeply in the context of another task. So I have no motivation to investigate it myself.

I added strace tool when we were switching to BSP 5.9, and I assisted @otyshchenko1 with testing. It was done with this commit:
https://github.com/xen-troops/meta-xt-prod-devel-rcar/commit/54034cc5cec55ab842c2f26ed21289c226627561

At those times I needed it. But afterward, there was no case when I used it. Thus, when it stopped working in Kirkstone the easiest way around was to simply remove it from the project definition.

So let's agree on an "issue on GitHub", which will be placed in this repo's backlog. Here it is:
https://github.com/xen-troops/meta-xt-prod-devel-rcar/issues/81

Reply:
"+"